Demo/Demo/DemoViewController.swift

```diff
@@ -148,22 +140,3 @@ public class DemoViewController<View: UIView>: UIViewController, Containable {
         }
     }
 }
-
-extension DemoViewController: CornerAnchoringViewDelegate {
```

Comment:
I did the name change, I considered making BaseDemoViewController into a private class but there are a few places where we are pushing controllers instead of using plain views, in those places we need to subclass BaseDemoViewController. 🤔

Reply:
Right, haven't thought about that.